modules/templates/util_string.go

```diff
@@ -36,3 +37,8 @@ func (su *StringUtils) Join(a []string, sep string) string {
 func (su *StringUtils) EllipsisString(s string, max int) string {
 	return base.EllipsisString(s, max)
 }
+
+func (su *StringUtils) ToInt(s string) int64 {
```

Comment:
we can remove it once we do time conversion not in backend but in frontend via JS and browser API etc ...